### PR TITLE
Added tuyagateway

### PR DESCRIPTION
**Pull request by jaburges**
Added Tuyagateway.xml and tuyagateway.png

### CODE REVIEW

**Comment by jrdnlc** [comment]
Switch base to testing please

**Comment by jaburges** [comment]
> Switch base to testing please
> 
> Edit: see other templates for xml format as an example. You have entries that are not needed

the Branch stuff? I can remove that

**Comment by jrdnlc** [comment]
`Timezone` is also not needed as Unraid adds that by default.

**Comment by jaburges** [comment]
what is best - to push the xml to you or to point in the devs repo?
https://github.com/TradeFace/tuyagateway/blob/development/etc/unraid_template_tuyagateway.xml

**Comment by Roxedus** [comment]
Re-add the branches :)

In the end the templateurl doesn't matter, however keep it as linking to this repo.

Don't set actual data in the config, lets just rely on defaults.

```xml
<Config Name="MQTT Host" Target="MQTTHOST" Default="127.0.0.1" Mode="" Description="MQTT Host IP" Type="Variable" Display="always" Required="true" Mask="false">127.0.0.1</Config>
```

can just be

```xml
<Config Name="MQTT Host" Target="MQTTHOST" Default="127.0.0.1" Mode="" Description="MQTT Host IP" Type="Variable" Display="always" Required="true" Mask="false"/>
```

File: templates/tuyagateway.xml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  <Privileged>false</Privileged>
+  <Support>https://github.com/tradeface/tuyagateway/wiki/docker</Support>
+  <Project>https://github.com/TradeFace/tuyagateway</Project>
+  <Overview>This is the official TuyaGateway 2.0 Docker</Overview>
+  <Description>TuyaGateway listens on MQTT topics and routes requests to your Tuya devices.</Description>
+  <Category>Tools: HomeAutomation: Other</Category>
+  <TemplateURL>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/tuyagateway.xml</TemplateURL>
+  <Icon>https://raw.githubusercontent.com/selfhosters/unRAID-CA-templates/master/templates/img/tuyagateway.png</Icon>
+  <Config Name="Config" Target="/usr/src/app/etc" Default="/mnt/user/appdata/tuyagateway" Mode="rw" Description="Container Path: /usr/src/app/etc" Type="Path" Display="advanced-hide" Required="true" Mask="false"></Config>
+  <Config Name="MQTT Host" Target="MQTTHOST" Default="127.0.0.1" Mode="" Description="MQTT Host IP" Type="Variable" Display="always" Required="true" Mask="false"></Config>
```
Comment (Roxedus) [inline templates/tuyagateway.xml:25]:
```suggestion
  <Config Name="MQTT Host" Target="MQTTHOST" Default="127.0.0.1" Mode="" Description="MQTT Host IP" Type="Variable" Display="always" Required="true" Mask="false"/>
```
For all the configs

**Review by Roxedus — CHANGES_REQUESTED** [review]

**Review by Roxedus — APPROVED** [review]

**Pull request merged**